iceoryx_posh/include/iceoryx_posh/internal/roudi_environment/roudi_environment.hpp

```diff
@@ -35,7 +35,7 @@ class RouDiEnvironment
 {
   public:
     RouDiEnvironment(const RouDiConfig_t& roudiConfig = RouDiConfig_t().setDefaults(),
-                     MonitoringMode monitoringMode = MonitoringMode::OFF,
+                     config::MonitoringMode monitoringMode = config::MonitoringMode::OFF,
```

Comment:
oh, just noticed that this will break the code of users who have their own RouDi extension ... but the new building blocks will break their code anyway

Reply:
I don't know if this is avoidable if we want to isolate toml and dynamic configuration code in a separate, lower safety target :confounded: .

Reply:
yes, but we break the API with the new building blocks anyway